iot-hub/Tutorials/Routing/SimulatedDevice/SimulatedDevice.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.*" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.*.0" />

```

Comment:
I've been changing our samples out from using this kind of version notation. We should explicitly update them, rather than take the chance a customer picks up the sample and has it fail because it gets one we haven't tested with. Sure, we shouldn't have breaking changes, but I don't want to stake a positive sample experience on that.

There's really no benefit to making them go to latest.